docs/source/backends.rst

```diff
@@ -0,0 +1,109 @@
+.. role:: hidden
+    :class: hidden-section
+
+torch.backends
+==============
+
+`torch.backends` controls the behavior of various backends that PyTorch supports.
+
+These backends include:
+
+- ``torch.backends.cuda``
+- ``torch.backends.cudnn``
+- ``torch.backends.mkl``
+- ``torch.backends.mkldnn``
+- ``torch.backends.openmp``
+- ``torch.backends.quantized``
+- ``torch.backends.xnnpack``
+
+
+torch.backends.cuda
+^^^^^^^^^^^^^^^^^^^
+
+.. autofunction::  torch.backends.cuda.is_built
+
+.. attribute::  torch.backends.cuda.matmul.allow_tf32
+
+    A :class:`bool` that controls whether TensorFloat-32 tensor cores are allowed for the
```

Comment:
"are allowed for the computation of matrix multiplications " -> "may be used in matrix multiplications on Ampere or newer GPUs. See :ref:\`tf32_on_ampere\`."

You don't need the second sentence of this paragraph.

Reply:
fixed